migrations/10_password_encoding.up.sql

```diff
@@ -0,0 +1,6 @@
+UPDATE users
+SET password = '$2a$10$vhlxLsj.HcuQVrbRTmOu6OOjTYK/ZeiLTKEA2Sf5JOaxZ98/KHGem'
```

Comment:
i think that we probably don't have to update default and superadmin via scripts cause they can be changed manually before.

Reply:
we need to reset it as well cause there is no possibility to restore default superadmin password